tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -12,6 +12,9 @@
 They are specified using the "fctype" parameter of each feature calculator, which is added using the
 set_property function. Only functions in this python module, which have a parameter called  "fctype" are
 seen by tsfresh as a feature calculator. Others will not be calculated.
+
+Feature calculators of type combiner should return the concatenated parameters sorted
```

Comment:
This is garantueed by https://github.com/blue-yonder/tsfresh/blame/54de174f593290a0c798ccad2ad7cad493c9628e/tsfresh/utilities/string_manipulation.py#L70, isn't?

Reply:
Well unfortunately no. Only if it is a non-combiner...